Examples/Run/Common/include/ActsExamples/Options/CommonOptions.hpp

```diff
@@ -41,7 +41,18 @@ void addMaterialOptions(boost::program_options::options_description& opt);
 void addInputOptions(boost::program_options::options_description& opt);
 
 /// Add common output-related options.
-void addOutputOptions(boost::program_options::options_description& opt);
+enum OutputFormat {
```

Comment:
Would you be willing to use an `enum class` here and provide the bit operators explicitly? I've done this for the `TrackPropMask` so you could copy that.

Reply:
I agree, it's always good to keep similar things implemented similarly.

Reply:
Sure! Its alway good to have strong typing. I will also think about a generalization...